src/main/java/shef/servlets/NewLiveStreamServlet.java

```diff
@@ -25,19 +25,18 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet responsible for creating new livestreams. */
+/** Servlet responsible for creating new LiveStream entities
+    and storing them in Datastore. */
 @WebServlet("/new-live-stream")
 public class NewLiveStreamServlet extends HttpServlet {
 
   @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String recipeKey = request.getParameter("recipe-selection");
-    String link = request.getParameter("live-stream-link");
-
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Entity liveStreamEntity = new Entity("LiveStream");
-    liveStreamEntity.setProperty("recipe-key", recipeKey);
-    liveStreamEntity.setProperty("link", link);
-
+    for (String param : request.getParameterMap().keySet()) {
+      liveStreamEntity.setProperty(param, (String) request.getParameterMap().get(param)[0]);
```

Comment:
can we avoid this map lookup by iterating over keys/values instead of just the key set.

Reply:
Tried to iterate over entries, but get an error because of the value type (String[]) - for-each not applicable to expression type
[ERROR]   required: array or java.lang.Iterable
[ERROR]   found:    java.util.Map<java.lang.String,java.lang.String[]>
Preferring this method over creating a bunch of unnecessary arrays so I can iterate over entries.